js/fc.js

```diff
@@ -103,8 +103,7 @@ var FC = {
         RC_tuning = {
             RC_RATE:         0,
             RC_EXPO:         0,
-            roll_pitch_rate: 0, // pre 1.7 api only
```

Comment:
Are you sure ? It is always set to 0, 
https://github.com/betaflight/betaflight-configurator/blob/master/js/msp/MSPHelper.js#L120